src/site/_drafts/perception-toolkit/index.md

```diff
@@ -0,0 +1,161 @@
+async function onMarkerChanges(event) {
+  event.preventDefault();
+  if (container. childNodes.length > 0) { return; }
+  const { found, lost } = event.detail;
+  // Deal with lost and found objects.
+}
+```
+
+Next, we show an appropriate card based on what the toolkit found. Notice the
+conveniences I've added to the user experience. Whether the marker is found or
+not, I provide one-click access to what I think is most useful in the
+circumstances.
+
+```js
+async function onMarkerChanges(event) {
+  event.preventDefault();
+  if (container. childNodes.length > 0) { return; }
+  const { found, lost } = event.detail;
+  const { Card } = PerceptionToolkit.Elements;
+  if (found.length === 0 && lost.length === 0) {
+    const card = new Card();
+    card.src = 'We wish we could help, but that\'s not our razor. Would you like to see <a href="/catalog">our catalog</a>?';
+    card.dataset.notRecognized = true;
+    container.appendChild(card);
+  } else if (found.length > 0 || lost.length > 0) {
+    const card = new Card();
+    card.src = found[0].content || lost[0].content;
+    // The ActionButton object is also part of the toolkit.
+    // Its instantiation is not shown.
+    carc.appendChild(reorderActionButton);
+    container.appendChild(card);
+  }
+}
+```
+
+Perception toolkit provides a built-in formatting for cards and buttons with the
+default stylesheet. But you can easily add your own. The provided `Card` and
+`ActionButton` object's contain `style` properties (among many others) that you
+can use to put your organizational stamp on the look and feel. To include the
+default stylesheet, add a link element to the pages that need it.
+
+```html
+<link rel="stylesheet" href="//path/to/toolkit/styles/perception-toolkit.css">
+```
+
+## Match structured data
+
+The toolkit can't find just any object in the real world. You must provide it
+with linked JSON data for the markers you want it to recognize. This data also
+contains information about those markers that will be shown to the user.
+Anything you add to a display card with card.src (see above) will be in addition
+to what's provided by the linked data.
+
+This data itself can either be in an external file or in the application page.
+Both cases use a <script> tag. If you're new to linked data, notice the mime
+type.
+
+```html
+<script type="application/ld+json" src="//path/to/your/sitemap.jsonld">
+```
+
+Or
+
+```js
+<script type="application/ld+json">
+{
+  "@context": "https://schema.googleapis.com/",
+  "@type": "ARArtifact",
+  "arTarget": {
+    "@type": "Barcode",
+    "text": "Inline Barcode"
+  },
+  "arContent": {
+    "@type": "WebPage",
+    "url": "http://example.com/item.html",
+    "name": "Item name",
+    "description": "A short description explaining a little more about the item.",
+    "image": "http://example.com/images/item-image.jpg"
+  }
+}
+</script>
+```
+
+## Conclusion
+
+As I said at the top, this is not an exhaustive look at the Perception Toolkit.
```

Comment:
Maybe this could be rephrased as:

"While this is not an exhaustive look at the Perception Toolkit, it hopefully gives you a sense of how easy it is to add visual searching to a web site. If you'd like to learn more check out the Getting Started guide and the demo. When you're ready, take a deeper dive into the Toolkit reference documentation to understand all that it can do."